packages/react/src/patterns/sub-patterns/Card/__stories__/Card.stories.js

```diff
@@ -91,34 +96,44 @@ storiesOf('Patterns (Sub-Patterns)|Card', module)
       '4:3': '4x3',
       '1:1': '1x1',
     };
+    const themes = {
+      white: '',
+      g10: 'g10',
+      g90: 'g90',
+      g100: 'g100',
+    };
+    const image = props.cardProps().image && {
+      defaultImage: 'https://picsum.photos/id/2/600/300',
+      alt: 'featured link image',
+    };
 
-    return !select('Ratio', ratio, ratio['none']) ? (
-      <Card
-        title={title}
-        copy={copy}
-        href={href}
-        cta={cta}
-        icon={ArrowRight20}
-        target={target}
-        type={type}
-      />
-    ) : (
-      <div
-        className={`bx--aspect-ratio bx--aspect-ratio--${select(
-          'Ratio',
-          ratio,
-          ratio['none']
-        )}`}>
-        <Card
-          title={title}
-          copy={copy}
-          cta={cta}
-          icon={ArrowRight20}
-          href={href}
-          target={target}
-          type={type}
-          className="bx--aspect-ratio--object"
-        />
+    return (
+      <div className={`bx--card--${select('theme', themes, themes.white)}`}>
+        <div className="bx--grid">
+          <div className="bx--row">
+            <div className="bx--col">
```

Comment:
This div is only causing problems. I'd remove it. Your columns are set with the div on line 115.